cf/api/service_keys.go

```diff
@@ -39,3 +42,24 @@ func (c CloudControllerServiceKeyRepository) CreateServiceKey(instanceId string,
 
 	return nil
 }
+
+func (c CloudControllerServiceKeyRepository) ListServiceKeys(instanceId string) (serviceKeys []models.ServiceKey, apiErr error) {
+	path := fmt.Sprintf("/v2/service_keys?q=service_instance_guid:%s", instanceId)
+
+	serviceKeys = []models.ServiceKey{}
+	apiErr = c.gateway.ListPaginatedResources(
+		c.config.ApiEndpoint(),
+		path,
+		resources.ServiceKeyResource{},
+		func(resource interface{}) bool {
+			serviceKey := resource.(resources.ServiceKeyResource).ToModel()
+			serviceKeys = append(serviceKeys, serviceKey)
+			return true
+		})
+
```

Comment:
since you are using named return vars, we can just have a statement `return` here, both `serviceKeys` and `apiErr` will be returned. However, like what I have mentioned in the other comments, in general we want to avoid naming our return vars.